### ISSUE TITLE
App isn't recognising headphones

### ISSUE THREAD

**Issue by hajkenek** [craigster1991/arctis-battery-win-app#11]
Hi,
App is showing only my mouse, no matter which port I use. Probably it's related to bad naming of Arctis 9 in USB. If it's helpful check this list.
Device is: Arctis 9 (probably first gen)
![image](https://user-images.githubusercontent.com/66490316/223787675-b24b3222-92d7-4903-b733-a74c9660076f.png)

**Comment by nshannon123** [comment]
Confirm this is happening with my Arctis 9's as well - VID is 1038 and PID is 12C4 in Device Manager

**Comment by hajkenek** [comment]
Okay, I see #19 is the same issue, so keep this closed.

**Comment by craigster1991** [comment]
Please try the latest version published today, it shows more debuigging information in the "cant see your deviuce" popup
https://github.com/craigster1991/arctis-battery-win-app/releases/tag/v0.2.2

**Comment by hajkenek** [comment]
Yep, there it is!
![image](https://github.com/craigster1991/arctis-battery-win-app/assets/66490316/e098922e-148a-4a57-af34-632fd3f17567)

**Comment by nshannon123** [comment]
Yep mine showing as well in the Don't see your device window now:

![image](https://github.com/craigster1991/arctis-battery-win-app/assets/8133661/933cf530-a0ef-4d0d-b9cc-175f21696079)

**Comment by tstaddon** [comment]
Here's the Arctis 9x ones if they'll help.

![image](https://github.com/craigster1991/arctis-battery-win-app/assets/51123466/1ac2241e-95d3-447c-ad51-6c616383eb3a)

If you need further details, this is from Device Manager when filtered by container...

![image](https://github.com/craigster1991/arctis-battery-win-app/assets/51123466/27f19a20-ef63-4879-b3ff-46dec9a5eceb)
